attestation-service/attestation-service/src/policy_engine/mod.rs

```diff
@@ -13,6 +15,40 @@ pub enum PolicyEngineType {
     OPA,
 }
 
+#[derive(Error, Debug)]
+pub enum OpaError {
+    #[error("Failed to create policy directory: {0}")]
```

Comment:
I prefer to add an empty line inbetween error enums

Reply:
Fixed the linter , This error enum will look pretty big if i add empty line between each error ? I see other files are not using empty lines as well, should we follow the same here ?

Reply:
Yeah, we can keep it as-is in this PR and re-organize that later altogether.